Contact/Block/Contact.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Gorilla\Contact\Block;
+
+class Contact extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @inheritdoc
+     */
+    public function _prepareLayout()
+    {
+        parent::_prepareLayout();
+        $this->pageConfig->getTitle()->set(__('Sign Up'));
```

Comment:
This can be done through layout XML. Not best practice to do this within a block class.